content/authoring/kata/create-first-kata.md

```diff
@@ -0,0 +1,256 @@
+
+
+### How to not step on the minefield of duplicate kata
+
+
+#### Search for keywords to see if someone has done the idea before
+
+As we all know, before you even engage your enemies, you need to send scouts forward.
+
+For example, while [Peano](https://www.codewars.com/kata/search/?q=peano) and [Church](https://www.codewars.com/kata/search/?q=church) numbers are definitely not easy, as the search results clearly show, they've been done many times already.
+
+In a way, knowing the kata in the wild is very similar to knowing your enemies: you get to see how others write the same kata and other people's (sometimes) brilliant solution in these kata.
+
+Then, maybe you can decide whether to retreat or make a **harder** version of the kata as a fallback if you feel the existing one is too lackluster/easy.
+
+
+#### Use a novel/original idea
+
+If you find yourself worrying about hitting duplicates all the time, try to push yourself to think out-of-the-box. Don't worry, coming up with good ideas is hard! But if you can ensure that your ideas are always novel and original, you can be almost certain nobody will have taken the flag before you.
+
+It also has a side benefit of making people think that you're clever, which will be reflected in the satisfaction rating.
+
+
+#### Solve more kata
+
+Naturally, when you've already solved 10 Fibonacci kata, solving another one will make you very nauseous, making you naturally allergic to duplicates ;-)
+
+
+#### Solve more kata (again)
+
+While the minimum honor for creating a kata is merely 300 honor, if you try to create a kata with just 300 honor, it'll most likely inadvertently be bad.
+
+Why? With just 300 honor, you haven't even got a grasp of what kata are actually like.
+
+Anyone can reach 300 honor in a very short time, from a few hours to a day or two. This is, obviously, far from enough. Thus, you need to train more.
+
+Solving more kata to gain more experience can help one significantly in the following aspects:
+
+- Getting more experienced will help you *understand* how hard your kata is, so that you can tune your kata to your desire easily.
+- As you train more, you tend to know what the most efficient solution for a kata is. This is crucial to performance and golf kata: you don't want to make a performance kata when you can only write sub-par solutions! You'll get pwned hard by veterans ;-)
+- Encountering kata issues and looking at comment sections will let you understand the common issues people will raise. Learn from history and don't make the same mistake again!
+- Looking at solutions (and more importantly, solution comment sections) will give you an insight into what constitutes good practice.
+- It also allows you to see how others write their tests. Writing good tests is **hard**, especially if your kata is also hard.
```

Comment:
Combine these.

Honestly, this section about avoiding duplicates is too long and doesn't contain much information. It just says to search with keywords, maybe make harder variants, come up an original idea somehow, and solve more to understand how things work.
This section doesn't fit under this "avoiding duplicates" heading and says to "learn from existing kata".

Reply:
Actually, I like how it stresses the need of getting more experience. But I agree this section has to be reduced, i will think of something.

Reply:
I tried to reorganize this section. Unfortunately, the final result did not end up smaller :(

Please let me know what you think, and as the next step I can try and get rid of redundancies. I'd like to do this _after_ merging though, if possible.